scss/_core.scss

```diff
@@ -17,3 +17,4 @@
 
 //Utiliticore/es
 @import "core/tokens/_sizing.scss";
+@import "core/tokens/emptystate"
```

Comment:
Empty-state do we want to keep in tokens or blocks?

Reply:
it doesn't seems like a token, it's a component itself

Reply:
Will move it to blocks